frontend/src/layouts/DashboardLayout/DashboardLayout.module.scss

```
@@ -0,0 +1,46 @@
+.main {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.layoutBody {
+  flex: 1;
+  display: flex;
+  min-height: 0;
+  align-items: stretch; 
+  min-height: 0; 
+}
+
+.sidebar {
+  flex: 0 0 100px;
+  height: 100%;
+  align-self: stretch;
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+}
+
+.content {
+  flex: 1;
+  min-width: 0;
+  display: flex;
+  flex-direction: column;
+}.main {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
```

Should there be a break line? Also, is it duplicated with line 1?
```suggestion
}

.main {
  min-height: 100vh;
  display: flex;
  flex-direction: column;
}
```